bindata/network/frr-k8s/OWNERS

```diff
@@ -0,0 +1,24 @@
+reviewers:
+  - abhat
```

Comment:
Probably use a OWNERS_ALIASES file to define an alias for the core CNO team so that we don't have to maintain the list in two different places

Reply:
Done, hope I am not locking ourselves out :D